src/chordChart.ts

```diff
@@ -513,25 +518,25 @@ module powerbi.extensibility.visual {
 
             switch (options.objectName) {
                 case "axis": {
-                    return ChordChart.enumerateAxis(settings);
+                    return ChordChart.enumerateAxis(settings, this.localManager);
                 }
                 case "dataPoint": {
-                    return ChordChart.enumerateDataPoint(settings, this.data.labelDataPoints);
+                    return ChordChart.enumerateDataPoint(settings, this.data.labelDataPoints, this.localManager);
                 }
                 case "labels": {
-                    return ChordChart.enumerateLabels(settings);
+                    return ChordChart.enumerateLabels(settings, this.localManager);
                 }
                 default: {
                     return [];
                 }
             }
         }
 
-        private static enumerateAxis(settings: IChordChartSettings): VisualObjectInstance[] {
+        private static enumerateAxis(settings: IChordChartSettings, localManager: ILocalizationManager): VisualObjectInstance[] {
             let axisSettings: IAxisSettings = settings.axis,
                 instances: VisualObjectInstance[] = [{
                     objectName: "axis",
-                    displayName: "Axis",
+                    displayName: localManager.getDisplayName("Visual_Axis"),
```

Comment:
JFYI, `displayName` is not shown by Power BI if `selection` is null or undefined.

No action requires here.